Tests/CommonTests/SerializationTests.cs

```diff
@@ -73,7 +73,7 @@ public void SerializeRules()
             Rules reloaded = Rules.Load(rulesFile);
 
             string expectedXmlContent = @"<?xml version=""1.0"" encoding=""utf-8""?>
-<rules xmlns:xsi=""http://www.w3.org/2001/XMLSchema-instance"" xmlns:xsd=""http://www.w3.org/2001/XMLSchema"">
+<rules xmlns:xsd=""http://www.w3.org/2001/XMLSchema"" xmlns:xsi=""http://www.w3.org/2001/XMLSchema-instance"">
```

Comment:
I wanted to convert this to a raw string literal but it requires C# 11 and I cannot update the pipeline.